src/handler-context/make-incoming-messages.ts

```diff
@@ -170,11 +169,16 @@ export const makeIncomingMessages = async <TBody extends HandlerMessageBody>(
     if (SQSMessageIncludesS3EventRecord(result)) {
       result = await makeS3MessagesFromSqsMessages(result, logger);
     }
-  } else {
+  } else if (isS3Event(event)) {
     result = {
       incomingMessages: await makeS3Messages(event),
       failedIds: [], // S3 events have no message IDs (throw error on failure instead)}
     };
+  } else {
+    return {
```

Comment:
Prob worth covering this with a test. Maybe this?

```
  describe("Incoming event that is not SQS or S3", () => {
    beforeEach(() => {
      testEvent = "a non sqs or S3 event" as any;
    });

    it("returns the event", async () => {
      const result = await makeIncomingMessages(
        testEvent,
        testIncomingMessageBodyTypeGuard as any,
        testLogger,
        testFailuresAllowed
      );

      expect(result).toEqual({
        incomingMessages: ["a non sqs or S3 event"],
        failedIds: [],
      });
    });
  });
```

Reply:
Done.